general/README.md

```diff
@@ -208,15 +209,102 @@ Avoid variable indirection when possible. Reassigning variables when no transfor
 ```js
 // good
 _handleEvent = function(e) {
-    _doSomethingWithEvent(e.target.checked);   
+    _doSomethingWithEvent(e.target.checked);
 }
 
 // bad
 _handleEvent = function(e) {
     var checked = e.target.checked;
 
-    _doSomethingWithEvent(checked);   
+    _doSomethingWithEvent(checked);
 }
 ```
 
 **[⬆ back to top](#table-of-contents)**
+
+## Naming Conventions
+> There are only two hard things in Computer Science: cache invalidation and naming things. *Phil Karlton*
+
+### Variables: Classes or Newable Functions (meant to be factories).
+It should be singular and begin with a capital letter.
+
+```js
+var ticket = new Ticket(id: 555);
+```
+### Variables: helpers and commmon variables
+
+It should always start with a lower case and be camel Case if is composed by 2 words.
+
+```js
+    var pagination = require('pagination');
+
+    pages = pagination.getPages();
+```
+
+### Varialbles: boolean
+
+It sholud start either with `is` or `has`
+
+```js
+var isAvailable = true,
+    hasTickets = false;
+```
+### Variables: jquery element or elements
+
+It sholud start with a dollar sign (`$`) so we can identify quickly by static analysis (not running and instecting the actuall code) that we are dealing with a jquery element/s.
```

Comment:
Variables that contain jQuery elements should start with a dollar sign (`$`) so that we can quickly identify these types of variables by scanning the code (i.e. static analysis). This makes code review easier.